scripts/release-stats.js

```diff
@@ -30,25 +27,21 @@ const getJSON = url =>
     ),
   );
 
-const question = query => {
+const question = async query => {
   const rl = readline.createInterface({
     input: process.stdin,
     output: process.stdout,
   });
-  return new Promise(resolve => rl.question(query, resolve)).then(response => {
-    rl.close();
-    console.log();
-    return response;
-  });
+  const response = await new Promise(resolve => rl.question(query, resolve));
+  rl.close();
+  console.log();
+  return response;
 };
 
-const confirm = str => !['n', 'no'].includes(str.toLowerCase());
-
 const prompt = async questions => {
   const results = {};
   for (const q of questions) {
-    const options = q.type === confirm ? '(Y/n) ' : '';
-    results[q.name] = await question(`${q.message} ${options}`).then(q.type);
+    results[q.name] = await question(q.message).then(q.type);
```

Comment:
Can I have the trailing space back? Looks a little nicer 😄

```suggestion
    results[q.name] = await question(`${q.message} `).then(q.type);
```